### DISCUSSION TITLE
Canno't confirm my student id verification

### DISCUSSION THREAD

**Discussion by lucaoskaique** [community/community#29544] [GitHub Education]
Hi there,

I just received the below email, telling me that I was going to start being billed for GitHub Pro if I didn't apply for a new coupon:

> Hey there,
> 
> 
> 
> Thanks for using GitHub! A while ago you used a GitHub coupon that was good for $4.00 off 2 years of GitHub Pro . This coupon will expire in 14 days (August 29, 2022) and your account will be downgraded to GitHub Free until you add a payment method.

I'm trying to validate it, but it asks to take a picture holding it and i'm using my phone because we are in 2022 and my university doesn't use paper anymore, so i'm using my phone with the file disponibilized in the university system.

![image](https://user-images.githubusercontent.com/20323741/184679526-c1bd1a9e-73b9-4607-a3e6-fa59de1c6fed.png)

when i'm trying to confirm i get this ERROR

![image](https://user-images.githubusercontent.com/20323741/184678680-8cb25974-5fc5-40c0-ba6b-09348b4debb1.png)

my university emmits a paper with real time validation in the system as it follows:

To validate access: http://academico.uergs.edu.br/validar and type the code ****************
(for security reasons im not gonna show the code)

How can you guys help me validate my student pack?

**Comment by mazkdevf** [comment]
Hello lucaoskaique!

Have you tried an image of the paper or document where it shows clearly the date?

**Reply by mazkdevf** [reply]
Like for example if you upload the PDF, or what ever the document is by format directly to the GitHub Submit an File

**Reply by lucaoskaique** [reply]
hi, mazk

i don't know if i'm not seeing it but i don't think there is this option to upload a file, i acctually thinks there SHOULD be this options, but not here

![image](https://user-images.githubusercontent.com/20323741/184681689-3cec4831-7dda-4f05-805e-5272d11a2957.png)

how should we proceed ?

**Reply by mazkdevf** [reply]
Hmm interesting, i did that today and it did have that 🤔

**Reply by lucaoskaique** [reply]
Even if we choose other options in "proof type" the "take picture" button doesn't change

**Reply by mazkdevf** [reply]
Well what happens if you try to fit it close as possible to webcam that it looks semi clear.
so the AI or human can see it clearly

**Reply by mazkdevf** [reply]
> 

Okay interesting 🤔

**Comment by lucaoskaique** [comment]
I'm still waiting for someone from @github-staff to help me, it's been 2 days already

**Reply by lucaoskaique** [reply]
i select to upload a file and it asks me to take a picure 
![image](https://user-images.githubusercontent.com/20323741/185209857-b4576ec2-c058-4226-b727-3507a4da4d6c.png)

**Reply by cHenrique0** [reply]
Hi. Did you solve your problem?
I used the Certificate of Enrollment too.
In **Proof Type**, choose the _Dated enrollment letter on school letterhead_ that should work.
Good luck!

**Reply by lucaoskaique** [reply]
noting works and we do not have any sopport from the team

![image](https://user-images.githubusercontent.com/20323741/185423773-70d264ee-d89c-4933-a7d6-57dd1a384f99.png)

**Comment by Peererxon** [comment]
same issue here

**Comment by Peererxon** [comment]
![image](https://user-images.githubusercontent.com/36285664/185281740-77cc9a4f-25ee-459e-bcf3-093d50ac8212.png)

**Comment by softwaredevelop** [comment]
same issue

> Your application cannot be reviewed until you fix the following:
> - Please modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.
> - The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

**Reply by martinsRossi** [reply]
Could you solve the "Please modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again." problem?

**Reply by softwaredevelop** [reply]
I modified browser settings and restart ..nothing. Login from my mobile ..problem solved. GitHub likes mobile platform.

**Reply by maheshcoder** [reply]
Sir i have same issue but still not working

**Reply by softwaredevelop** [reply]
Try login in mobile. In my desktop browser are vpn, tor, etc..,  and github don't like these because geolocation.
Good Luck!

**Reply by cbird808** [reply]
same problem on my laptop, win10, firefox

```
Your application cannot be reviewed until you fix the following:

    Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.
```

I did enable location access...

**Reply by cbird808** [reply]
it asks to access my camera, rather than my location, when I submit my app

**Reply by cbird808** [reply]
in chrome, if you right click on the lock icon next to the web url and select site settings, you can specify that location access is allowed.  Even with that, I still am getting the location error

**Reply by cbird808** [reply]
same problem on my phone

**Reply by kravrolens** [reply]
<img width="742" alt="image" src="https://user-images.githubusercontent.com/57486569/195800639-92e21275-4e7e-417c-963a-efe8da457237.png">

Same problem on my desktop and phone. I have completed user profile with my full name and a short bio, location access was allowed and VPN was shut down.

**Reply by egyangel** [reply]
same problem on my phone and pc

**Reply by kravrolens** [reply]
> Try login in mobile. In my desktop browser are vpn, tor, etc.., and github don't like these because geolocation.
> Good Luck!

thank goodness, github prefers mobile platform.

**Reply by cbird808** [reply]
I opened a ticket with github.  It wasn't easy to find, so here's the link: https://support.github.com/contact/education?tags=hubberfy_education

**Reply by TinPoChan** [reply]
Getting the location error as well, can't get rid of it no matter what browsers I'm using... Github c'mon...

**Reply by egyangel** [reply]
@cbird808  any news ? 
 thanks

**Reply by kajinmo** [reply]
On the computer, I turned off antivirus, antimalware and Adblock. I authorized the location and tried in three browsers. But I still got meaningless error messages ("Cannot find location / No dated school ID").

Following the advice of the colleagues above, I tried on my cell phone:

1) Turn on mobile GPS and enable mobile browser pop-up notification
2) Go to https://education.github.com/pack link and tried again
3) Even if you try on the phone, it will continue the process you were trying on the computer
4) Try again to get the error message and the Cancel button next to "Process my application")
5) Click Cancel
6) You will return to the academic email selection step. Redo the whole process (at this step a popup appeared asking if the site could access my location)
If uploading the certificate is not working, redo the whole process and this time try to take a picture of the monitor.
8) Click "Process My Application" and voilá

-----

Português/BR

No computador, desliguei antivirus, antimalware e Adblock. Autorizei a localização e tentei em três navegadores. Mas mesmo assim recebia mensagens de erros sem sentido ("Não é possível encontrar localização / Sem identificação escolar datada").

Seguindo a dica dos colegas acima, tenti no celular:

1) Ligue o GPS do celular e ative notificação de pop-up do navegador do celular
2) Vá ao link https://education.github.com/pack e tentei de novo
3) Mesmo você tentando no celular, ele vai dar continuidade no processo que você estava tentando no computador
4) Tente de novo para aparecer a mensagem de erro e o botão Cancel do lado do "Process my application")
5) Clique em Cancelar
6) Você vai voltar à etapa de seleção do e-mail acadêmico. Refaça todo o processo (nesta etapa apareceu um pop-up perguntando se o site poderia acessar minha localização)
7) Se fazer upload do certificado não está dando certo, refaça todo o processo e dessa vez tente tirar uma foto do monitor.
8) Clica em "Process my application" e voilá

**Reply by amanuel-hailu** [reply]
Thank you, I was able to get my application working!

**Comment by cbird808** [comment]
Github auto replied to my ticket and closed it. On one hand, I'm still getting emails telling me my academic status is going to expire. Now when I click the link to update my status, it says my status was last confirmed on Oct 12, 2022.

I'm going to wait for it to break and cause a disaster in my class

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by thiagoedson** [comment]
Não funciona 
Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.

**Comment by Ajay-singh1** [comment]
If you have your School ID or Enrollment Letter You can upload it. Github wont give you access because the screenshot cn be of different school.

**Comment by kajinmo** [comment]
On the computer, I turned off antivirus, antimalware and Adblock. I authorized the location and tried in three browsers. But I still got meaningless error messages ("Cannot find location / No dated school ID").

Following the advice of the colleagues above, I tried on my cell phone:

1) Turn on mobile GPS and enable mobile browser pop-up notification
2) Go to https://education.github.com/pack link and tried again
3) Even if you try on the phone, it will continue the process you were trying on the computer
4) Try again to get the error message and the Cancel button next to "Process my application")
5) Click Cancel
6) You will return to the academic email selection step. Redo the whole process (at this step a popup appeared asking if the site could access my location)
If uploading the certificate is not working, redo the whole process and this time try to take a picture of the monitor.
8) Click "Process My Application" and voilá

-----

Português/BR

No computador, desliguei antivirus, antimalware e Adblock. Autorizei a localização e tentei em três navegadores. Mas mesmo assim recebia mensagens de erros sem sentido ("Não é possível encontrar localização / Sem identificação escolar datada").

Seguindo a dica dos colegas acima, tenti no celular:

1) Ligue o GPS do celular e ative notificação de pop-up do navegador do celular
2) Vá ao link https://education.github.com/pack e tentei de novo
3) Mesmo você tentando no celular, ele vai dar continuidade no processo que você estava tentando no computador
4) Tente de novo para aparecer a mensagem de erro e o botão Cancel do lado do "Process my application")
5) Clique em Cancelar
6) Você vai voltar à etapa de seleção do e-mail acadêmico. Refaça todo o processo (nesta etapa apareceu um pop-up perguntando se o site poderia acessar minha localização)
7) Se fazer upload do certificado não está dando certo, refaça todo o processo e dessa vez tente tirar uma foto do monitor.
8) Clica em "Process my application" e voilá
